be/src/exec/pipeline/scan_operator.h

```diff
@@ -55,6 +57,7 @@ class ScanOperator final : public SourceOperator {
     const vectorized::RuntimeFilterProbeCollector& _runtime_filters;
     PriorityThreadPool* _io_threads = nullptr;
     OptionalChunkSourceFuture _pending_chunk_source_future;
+    std::atomic<int>* _active_scan_operators_count;
```

Comment:
_active_scan_operators_count is needless, a driver never be accessed in two or more threads,  may be u want close ScanOperator only once, a common field _is_closed field can work fine.